README.md

```diff
@@ -1,3 +1,5 @@
+/*
```

Comment:
While wrapping the readme in a comment does technically make this rust, it breaks existing functionality (like many other proposed rust rewrites).

Perhaps you can use some macro-magic so that this remains valid markdown while also being able to be run as a rust program?